rllib/core/learner/learner.py

```diff
@@ -55,6 +55,8 @@
 from ray.rllib.utils.framework import try_import_tf, try_import_torch
 from ray.rllib.utils.metrics import (
     ALL_MODULES,
+    DATASET_NUM_ITERS_PER_LEARNER_TRAINED,
```

Comment:
2 dumb questions:

* Will we put all OfflineData results into a subkey: `offline_data`? Similar to `env_runners` or `learners`?
* If yes, should we rename these into `NUM_DATASET_ITERS_...`
* I feel like we shouldn't do this per Learner, but rather per main Algorithm iteration. Meaning, we log/create the metric as `reduce=sum, clear_on_reduce=True` (`False` for the lifetime one) and then report it as a sum across all Learners. This would then be analogous to the online stats, e.g. `NUM_ENV_STEPS_TRAINED` and `NUM_ENV_STEPS_TRAINED_LIFETIME`. What do you think?

Reply:
Isn't this how this will be done? These metrics are used inside the `Learner`s and then reported to the main Algorithm that merges the dicts from all Learners. This metric should keep track how many updates a learner indeed took and if there is a divergence between the desired and the taken number of updates.